### PR TITLE
fix if ms:character in describfeaturetype

### PR DESCRIPTION
**Pull request by julsbreakdown**
@sbrunner

### CODE REVIEW

File: geoportal/c2cgeoportal_geoportal/views/entry.py [hunk 1]
```diff
@@ -1484,10 +1484,10 @@ def themes(self):
                                 if 'alias' in c.attrib:
                                     attrib[c.attrib['name']] = c.attrib['alias']
                             types[child.attrib['name']] = attrib
-
                     attributes = {}
                     for name, type_ in elements.items():
-                        attributes[name] = types[type_]
+                        if type_ in types:
+                            attributes[name] = types[type_]
```
Comment (sbrunner) [inline geoportal/c2cgeoportal_geoportal/views/entry.py]: I think that we should add a warning message ...

Comment (julsbreakdown) [reply geoportal/c2cgeoportal_geoportal/views/entry.py]: in the console?

Comment (sbrunner) [reply geoportal/c2cgeoportal_geoportal/views/entry.py]: Yes :-)

Comment (sbrunner) [reply geoportal/c2cgeoportal_geoportal/views/entry.py]: In the else :-)

File: geoportal/c2cgeoportal_geoportal/views/entry.py [hunk 2]
```diff
@@ -1484,10 +1484,13 @@ def themes(self):
...
+                        else:
+                            log.warning("{} does not exist in {}, this is probably due"
+                            "to the use of ms:character in the describeFeatureType".format(type_, types))
```
Comment (sbrunner) [inline geoportal/c2cgeoportal_geoportal/views/entry.py]:
```
log.warning(
    "The provided type '{}' does not exist, available types: {}.".format(
        type_, ', '.join(types.keys())
    )
)
```

Comment (julsbreakdown) [reply geoportal/c2cgeoportal_geoportal/views/entry.py]: No mention about ms:character?

**Pull request merged**